omero/sysadmins/troubleshooting.txt

```diff
@@ -344,18 +346,19 @@ three things to check:
 
 In the final case, two issues arise. Firstly, when performing some operations,
 the clients make use of temporary file storage and log directories, as
-described in the client configuration section of :doc:`system-requirements`.
+described in the :ref:`client_directories` section of
+:doc:`system-requirements`.
 If your home directory is stored on a network, possibly NFS mounted (or
 similar), then these temporary files are being written and read over the
 network which can slow access down. Secondly, the OMERO.server also accesses
-the ``$HOME/omero/tmp`` and ``$HOME/omero/log`` folders of the user the server
-process is running as. As the server makes heavier use of these folders than
-the clients, if the OMERO.server user's home(~) is stored on a network drive
-then slow performance can occur.
+the temporary and log folders of the user the server process is running as. As
+the server makes heavier use of these folders than the clients, if the
+OMERO.server user directory is stored on a network drive then slow performance
+can occur.
 
-To resolve this, you can define an :envvar:`OMERO_TEMPDIR` environment
+To resolve this, you can define an :envvar:`OMERO_TMPDIR` environment
 variable pointing to a temporary directory located on the local file system
-(e.g. `/tmp/`).
+(e.g. :file:`/tmp/omero_tmp`).
```

Comment:
Is the `_tmp` really required here?  Is it not implied that it's temporary by the fact it's localed in `/tmp/`?